app/retail/utils.py

```diff
@@ -356,14 +353,14 @@ def get_bounty_median_turnaround_time(func='turnaround_time_started', keyword=No
 
 def get_bounty_history(keyword=None, cumulative=True):
     bh = [
-        ['', 'Bounties', 'Tips', 'Grants', 'Kudos', 'Ads', 'Ecosystem'],
+        ['', 'Hackathons', 'Grants', 'Other'],
     ]
     initial_stats = [
-        ["December 2017", 5534, 2011, 0, 0, 0, 0],
-        ["January 2018", 15930, 5093, 0, 0, 0, 0],
-        ["February 2018", 16302, 7391, 0, 0, 0, 0],
-        ["March 2018", 26390, 8302, 0, 0, 0, 0],
-        ["April 2018", 37342, 10109, 0, 0, 0, 0],
+        ["December 2017", 5534 + 2011, 0, 0],
```

Comment:
A note to anyone else reviewing this, I actually think leaving it like this instead of doing the calculations and writing the sum directly is a great idea, because this shows where the number is coming from and what the history of this change is. Very slight performance hit vs more understandable code is usually a good idea.